src/main/java/kdt/prgrms/kazedon/everevent/service/converter/ReviewConverter.java

```diff
@@ -52,8 +51,11 @@ public UserReviewReadResponse convertToUserReviewReadResponse(Page<UserReview> u
   }
 
   private List<String> convertToPictureUrls(String pictureUrl) {
-    return StringUtils.isNullOrEmpty(pictureUrl) ? new ArrayList<>()
-        : new ArrayList<>(List.of(pictureUrl));
+    return isBlankPictureUrl(pictureUrl) ? Collections.emptyList() : List.of(pictureUrl);
+  }
+
+  boolean isBlankPictureUrl(String pictureUrl) {
+    return pictureUrl == null || pictureUrl.isBlank();
```

Comment:
괄호쓰는건 어떤가요?ㅎㅎ 개취입니다..ㅎㅎㅎ

```suggestion
    return (pictureUrl == null) || (pictureUrl.isBlank());
```

Reply:
Good!! 수정하겠숩니다 ㅎㅎ